app/controllers/api/v1/lois_controller.rb

```diff
@@ -65,7 +69,28 @@ def create
       end
 
       if @loi.submitted
-        Mail.new( :to => @loi.email, :from => 'colleen@svpchicago.org', :subject => 'Submission to SVP received', :body => File.read('app/views/submission_email.html.erb')).deliver!
+        Mail.new( 
+          :to => @loi.email, 
+          :from => 'colleen@svpchicago.org', 
+          :subject => 'Submission to SVP received', 
+          :body => File.read('app/views/submission_email.html.erb'),
+          :content_type => 'text/html; charset=UTF-8'
+        ).deliver!
+
+        # The link that gets generated will need to be changed when deploying to production
+        File.open('app/views/loi_submitted_email_to_admin.html.erb', 'w') { |file| file.write(
+        "<p>A user has submitted an LOI form</p>
+        <p>Click below to see it</p>
+        <p><a href='https://demo-svp-chicago.herokuapp.com/lois/#{@loi.id}'>View LOI</a></p>"
```

Comment:
Great work.
I think here, there's a way to dynamically set the base url so it can be something like `BASE_URL + "/lois/#{@loi.id}`  
I'm not sure how, but I'll look into it.